src/wallet/db.cpp

```diff
@@ -129,7 +129,7 @@ bool BerkeleyEnvironment::Open(bool retry)
     if (fDbEnvInit)
         return true;
 
-    boost::this_thread::interruption_point();
+    std::this_thread::yield();;
```

Comment:
In Bitcoin, this code removed(https://github.com/bitcoin/bitcoin/pull/17405).

Reply:
bitcoin doesn't use `this::thread::yield()` anywhere!  its all removed from tapyrus now